tests/run.rs

```diff
@@ -139,7 +139,8 @@ mod yaml_parse_errors {
             result,
             format!(
                 "error in {}.protocols.yaml: \
-                 expected: array, got: Integer(42)",
+                 expected: array, \
+                 got: Node(Integer(42), Some(Marker {{ index: 10, line: 1, col: 10 }}))",
```

Comment:
I don't think this test should be changed in this PR.

Reply:
Ok, I changed this back. But I think it would be useful to include the source location in this error message in a future change.